### ISSUE TITLE
How about training with 8-bit jpg image?

### ISSUE THREAD

**Issue by ArsenLuca** [cchen156/Learning-to-See-in-the-Dark#1]
I just tried to train Sony dataset after converting the raw data into jpg image and found the result quite terrible. So are there any tricks when training 8bit data?
P.S.:
     The  channel number of input is 12, which results from packing each channel of rgb into four.

**Comment by ArsenLuca** [comment]
Moreover, how did you pack the rgb data  as input in the experiment?

**Comment by N2ITN** [comment]
I tried importing PIL and loading a JPG, converting into RGBA, and putting it through the machinery, and got weird results. I think it's because the algorithm is designed to work on Bayer images (the arrangement of color filters on the pixel array of an image sensor), which is structurally different from a JPG data type.

I'm not sure if there is a way to "Bayerize" the JPG or if that is out of the question due to the irreversable (lossy) JPG compression.

I tried this on a RAW image from my Google Pixel instead and didn't get great results. I think adapting this process to different camera requires changing some of the constants around picture resolution and the values used when subtracting the black. Additionally as stated in the paper, without a baseline image, the amplification ratio will be unknown.

I'm not exactly an expert on this topic, but these are my thoughts after spending a couple hours trying to SIR some of my own photos.
